src/mapper.rs

```diff
@@ -146,15 +146,29 @@ impl<'s> ProguardMapper<'s> {
                                 None => (line_mapping.startline, Some(line_mapping.endline)),
                             }
                         });
-                    let members = class.members.entry(obfuscated).or_insert_with(Vec::new);
-                    members.push(MemberMapping {
-                        startline,
-                        endline,
-                        original_class,
-                        original,
-                        original_startline,
-                        original_endline,
-                    });
+                    class
+                        .members
+                        .entry(obfuscated)
+                        .and_modify(|members| {
+                            members.push(MemberMapping {
+                                startline,
+                                endline,
+                                original_class,
+                                original,
+                                original_startline,
+                                original_endline,
+                            })
+                        })
+                        .or_insert_with(|| {
+                            vec![MemberMapping {
+                                startline,
+                                endline,
+                                original_class,
+                                original,
+                                original_startline,
+                                original_endline,
+                            }]
```

Comment:
This also made a small dent in performance. The point here is to create the `vec` with capacity for the initial entry to avoid additional memory allocation when calling push. Not sure if there's some way to avoid declaring the `MemberMapping` struct twice.

Reply:
When I run the benchmark, it's saying that using `vec![]` is ~5% faster than `Vec::with_capacity(1)`. I'm not quite sure what exactly is different in the 2 methods though.